.github/workflows/test.yml

```diff
@@ -137,11 +137,6 @@ jobs:
         sudo apt-get clean
     - name: Install ninja
       uses: seanmiddleditch/gha-setup-ninja@master
-    - name: Install libxml2 and libxslt dev (pypy3)
-      if: ${{ startsWith(matrix.os,'ubuntu-') && ( matrix.python-version == 'pypy3' ) }}
-      run: |
-        sudo apt-get install -y libxml2-dev libxslt-dev
-        sudo apt-get clean
```

Comment:
Why was this deleted? If it isn't necessary, why was it added previously?

Reply:
While trying to get pyinstaller working on pypy I found out that this wasn't active since #550. With this PR pypy3 was replaced by pypy-3.7 which deactivates this step.